processing/src/main/java/io/druid/segment/virtual/SingleStringInputCachingExpressionColumnValueSelector.java

```diff
@@ -133,6 +133,12 @@ private ExprEval eval()
     return expression.eval(bindings);
   }
 
```

Comment:
Checks in this class?

Reply:
ExprEval already have checks.

Reply:
Please add comments for readers, expecting asserts

Reply:
added.